### DISCUSSION TITLE
How to port my 1+ year old config to the newer modular layout

### DISCUSSION THREAD

**Discussion by themaster567** [qtile/qtile#3572] [Q&A] [answered]
I'm wanting to get back into using Qtile after an extended time away from it. I want to recreate my old setup, but [my config](https://git.blairtech.org/killbotvii/t480-dotfiles/src/branch/master/.config/qtile/config.py) is completely different from how it's set up now. Everything used to be put into a single file, but now it's all split up into different modules and I'm not sure what code to put where. Would somebody be able to point me in the right direction to get started?

**Comment by m-col** [comment, marked as answer]
Try running `qtile migrate`, you may need to install the `bowler` library but that will catch some things that need to update. Reading through your config, the only things that I can see need changing are:

- `currentWindow` in some places to `current_window` (though you do already have `current_window` elsewhere!)
- likewise for `currentGroup`
- `from libqtile.command import lazy ` is not needed (but you have `from libqtile.lazy import lazy` already)
- also btw `main = None` is no longer needed.

I can't tell what would cause issues though. What issues do you see?

**Reply by themaster567** [reply]
Thanks for all the corrections! Later today or tomorrow I'll go through and run the migration tool and then make the changes you pointed out.

>What issues do you see?

No issues because I haven't started yet! 😄
